Dockerfile

```diff
@@ -2,32 +2,35 @@ FROM debian:jessie
 MAINTAINER maintainer@codeship.com
 
 RUN apt-get update && \
-    apt-get install -y locales && \
+    apt-get install -y locales wget && \
     rm -rf /var/lib/apt/lists/* && \
     localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 ENV LANG en_US.utf8
 
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+
+RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' >> /etc/apt/sources.list.d/pgdg.list
+
 RUN apt-get update && \
-    apt-get install -y postgresql-9.4 postgresql-contrib-9.4 && \
+    apt-get install -y postgresql-9.5 postgresql-contrib-9.5 && \
```

Comment:
We could define a `PG_VERSION` environment variable at the beginning and use the variable in the `RUN` statements. Would be easier to upgrade versions in the future.